ui/app/controllers/clients/index.js

```diff
@@ -57,47 +54,42 @@ export default Controller.extend(Sortable, Searchable, {
     { key: 'initializing', label: 'Initializing' },
     { key: 'ready', label: 'Ready' },
     { key: 'down', label: 'Down' },
+    { key: 'ineligible', label: 'Ineligible' },
+    { key: 'draining', label: 'Draining' },
   ]),
 
   optionsDatacenter: computed('nodes.[]', function() {
     const datacenters = Array.from(new Set(this.nodes.mapBy('datacenter'))).compact();
 
     // Remove any invalid datacenters from the query param/selection
     scheduleOnce('actions', () => {
-      this.set(
-        'qpDatacenter',
-        serialize(intersection(datacenters, this.selectionDatacenter))
-      );
+      this.set('qpDatacenter', serialize(intersection(datacenters, this.selectionDatacenter)));
     });
 
     return datacenters.sort().map(dc => ({ key: dc, label: dc }));
   }),
 
-  optionsFlags: computed(() => [
-    { key: 'ineligible', label: 'Ineligible' },
-    { key: 'draining', label: 'Draining' },
-  ]),
-
   filteredNodes: computed(
     'nodes.[]',
     'selectionClass',
     'selectionStatus',
     'selectionDatacenter',
-    'selectionFlags',
     function() {
       const {
         selectionClass: classes,
         selectionStatus: statuses,
         selectionDatacenter: datacenters,
-        selectionFlags: flags,
       } = this;
 
-      const onlyIneligible = flags.includes('ineligible');
-      const onlyDraining = flags.includes('draining');
+      const onlyIneligible = statuses.includes('ineligible');
+      const onlyDraining = statuses.includes('draining');
+
+      // “flags” were formerly a separate filter, now combined with statuses
+      const trueStatuses = statuses.without('ineligible').without('draining');
```

Comment:
an awkward name, I’m open to suggestions 😬

Reply:
My biggest complaint with this name is that I totally misinterpreted it at first. My read was "these are the statuses which are true", not "these are the values that are actually statuses".

Maybe `clientStatuses`, `nodeStatuses`, `statusValues`, or a more radical approach: rename the facet to `states` and make this `statusStates`.